prow/jobs/incubator/documentation-component/documentation-component.yaml

```diff
@@ -0,0 +1,37 @@
+job_template: &job_template
+  skip_report: false
+  always_run: true
+  decorate: true
+  path_alias: github.com/kyma-incubator/documentation-component
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181130-b28250b
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-incubator/documentation-component"
+        resources:
+          requests:
+            memory: 1.5Gi
+            cpu: 0.8
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+
+presubmits: # runs on PRs
+  kyma-incubator/documentation-component:
+    - name: pre-master-documentation-component
```

Comment:
Hm...maybe we can simplify naming and create it like for marketplaces repository?